### ISSUE TITLE
[Feature Request] Alpha or Beta release for Windows, please

### ISSUE THREAD

**Issue by b-morgan** [cnr-isti-vclab/meshlab#282]
I has been over a year since the last release of this project. There has, however, been a lot of commits since that time. For most Linux users, the tools (and scripts?) necessary to build a newer version are available. For most Windows users, the necessary knowledge, tools, (and scripts) are not as easy to obtain. Can someone with the necessary stuff create an Alpha or Beta release and make it available, please?

Once every couple of months would be significantly better that 15 months.

**Comment by mcallieri** [comment]
We are sorry. Things got in the way :( and the new release has been delayed way too much.
We are setting up an automatic build system, that should be alble to create "portable" folders with the new release.
We will post some news, hopefully, soon.

**Comment by b-morgan** [comment]
Thanks for the update. I believe most people can be patient if they see some progress even if its just a status update once in a while. I patiently await more news!

**Comment by mcallieri** [comment]
Hi... We have reached a somehow stable state.
The current WIN64 build can ba found here

https://ci.appveyor.com/project/cignoni/meshlab/build/artifacts

It is not an installer, just a zipped folder with a "portable" executable. We are starting testing, now, while fixing few more issues. We have used on some machines, here, and seems working. I have no idea, however if it has ALL the possible dll and Runtime that may be needed.
